services/core-api/app/api/mines/reports/models/mine_report.py

```diff
@@ -129,66 +128,58 @@ def mine_report_status_description(cls):
-            EmailService.send_template_email(subject, ms_recipients, ms_email_body, email_context, cc=None)
+        core_recipients, ms_recipients = self.collectRecipients(is_proponent)
 
-        else:
-            recipients = [self.mine.region.regional_contact_office.email, MDS_EMAIL]
-            if self.mine.major_mine_ind:
-                recipients = [MAJOR_MINES_OFFICE_EMAIL, MDS_EMAIL]
+        due_date = due_date = (self.due_date).strftime("%b %d %Y") if self.due_date else "N/A"
+        core_report_page_link =  f'{Config.CORE_PRODUCTION_URL}/mine-dashboard/{self.mine.mine_guid}/required-reports/{report_code.lower()}-required-reports'
+        ms_report_page_link = f'{Config.MINESPACE_PRODUCTION_URL}/mines/{self.mine.mine_guid}/reports'
 
-            subject_verb = 'Updated' if is_edit else 'Submitted'
-            subject = f'Code Required Report {subject_verb} for {self.mine.mine_name}'
+        if is_crr:
+            report_type = "Code Required Report"
+            compliance_details = self.mine_report_definition.compliance_articles[0]
+            report_name = f'{compliance_details.section}.{compliance_details.sub_section}.{compliance_details.paragraph} - {compliance_details.description}'
+            core_recipients.extend(self.getReportSpecificEmailsByReportType(compliance_details))
 
-            body_verb = 'uploaded document(s) to' if is_edit else 'submitted'
-            body = f'<p>{self.mine.mine_name} (Mine no: {self.mine.mine_no}) has {body_verb} a "{self.mine_report_definition_report_name}" report.</p>'
-
-            link = f'{Config.CORE_PRODUCTION_URL}/mine-dashboard/{self.mine.mine_guid}/reports/code-required-reports'
-            body += f'<p>View updates in Core: <a href="{link}" target="_blank">{link}</a></p>'
-            EmailService.send_email(subject, recipients, body)
+        else: #PRR
+            report_type = "Permit Required Report"
+            report_name = self.report_name
+            regional_email = self.mine.region.regional_contact_office.email
+            if self.mine.major_mine_ind:
+                core_recipients.append(PERM_RECL_EMAIL)
+            else:
+                core_recipients.append(regional_email)
+
+        email_context = {
+          "report_submision": {
+              "mine_number": self.mine.mine_no,
+              "mine_name": self.mine.mine_name,
+              "report_name": report_name,
+              "report_type": report_type,
+              "report_compliance_year": self.submission_year,
+              "report_due_date": due_date,
+              "report_recieved_date": (self.received_date).strftime("%b %d %Y at %I:%M %p"),
+          },
+          "minespace_login_link": Config.MINESPACE_PRODUCTION_URL,
+          "core_report_page_link": core_report_page_link,
+          "ms_report_page_link": ms_report_page_link
+        }
+
+        trigger_notification(f'Your ({self.mine_report_definition_report_name}) report has been recieved',
```

Comment:
I think this should be different text for CORE recipients

Reply:
updated